google-datacatalog-mysql-connector/ingest_cnfg.yaml

```diff
@@ -0,0 +1,23 @@
+---
+# By default, the connector retrieves the following metadata from a MySQL database:
+# table schema,
+# table name,
+# table type,
+# creation time,
+# time of the last update,
+# table size in Mb,
+# names of columns,
+# column default value,
+# information whether column nullable
+# column data type,
+# max length of char values in a column
+# and numeric precision of values in a column.
+#
+# Using this configuration file, you can turn on or off the scraping of other pieces of data.
+
+# Runs ANALYZE command for each user table, to update metadata tables and make database statistics more accurate.
+# Note, that it might take long time to ANALYZE big tables
+update-metadata: true
```

Comment:
Regarding case: noted;
regarding "run_analyze_table" - I'd like to have something more universal, because ANALYZE is not in SQL standard. So if in some DB the command not called ANALYZE, we still can use this same option in config file, without confusing the user. What about "refresh_metadata_info" or "refresh_metadata_tables"?